```
/home/prow/go/bin/golangci-lint run --modules-download-mode=readonly || (echo 'Run "make update-lint"' && exit 1)
142
pkg/apiserver/installer/installer.go:202:9: inline: Constant reflect.Ptr should be inlined (govet)
143			
case reflect.Ptr:
```

--- a/pkg/apiserver/installer/installer.go
+++ b/pkg/apiserver/installer/installer.go
@@ -199,7 +199,7 @@ func addObjectParams(ws *restful.WebService, route *restful.RouteBuilder, obj in
 			}
 			switch sf.Type.Kind() {
 			case reflect.Interface, reflect.Struct:
-			case reflect.Ptr:
+			case reflect.Pointer:
 				// TODO: This is a hack to let metav1.Time through. This needs to be fixed in a more generic way eventually. bug #36191
 				if (sf.Type.Elem().Kind() == reflect.Interface || sf.Type.Elem().Kind() == reflect.Struct) && strings.TrimPrefix(sf.Type.String(), "*") != "metav1.Time" {
 					continue

